tests/flows/staking/poa_validator_manager.go

```diff
@@ -0,0 +1,206 @@
+package staking
+
+import (
+	"context"
+	"math/big"
+	"time"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/crypto/bls"
+	"github.com/ava-labs/subnet-evm/accounts/abi/bind"
+	"github.com/ava-labs/subnet-evm/core/types"
+	"github.com/ava-labs/teleporter/tests/interfaces"
+	"github.com/ava-labs/teleporter/tests/utils"
+	"github.com/ethereum/go-ethereum/crypto"
+	. "github.com/onsi/gomega"
+)
+
+/*
+* Register a PoA validator manager on a L1. The steps are as follows:
+* - Generate random address to be the owner address
+* - Fund native assets to the owner address
+* - Deploy the PoAValidatorManager contract
+* - Attempt to initiate with non owner and check that it fails
+* - Initiate validator registration
+* - Deliver the Warp message to the P-Chain (not implemented)
+* - Aggregate P-Chain signatures on the response Warp message
+* - Deliver the Warp message to the L1
+* - Verify that the validator is registered in the validator manager contract
+
+* Delists the validator from the L1. The steps are as follows:
+* - Attempt to initiate with non owner and check that it fails
+* - Initiate validator delisting
+* - Deliver the Warp message to the P-Chain (not implemented)
+* - Aggregate P-Chain signatures on the response Warp message
+* - Deliver the Warp message to the L1
+* - Verify that the validator is delisted from the validator manager contract
```

Comment:
```suggestion
 * Register a PoA validator manager on a L1. The steps are as follows:
 * - Generate random address to be the owner address
 * - Fund native assets to the owner address
 * - Deploy the PoAValidatorManager contract
 * - Attempt to initiate with non owner and check that it fails
 * - Initiate validator registration
 * - Deliver the Warp message to the P-Chain (not implemented)
 * - Aggregate P-Chain signatures on the response Warp message
 * - Deliver the Warp message to the L1
 * - Verify that the validator is registered in the validator manager contract

 * Delists the validator from the L1. The steps are as follows:
 * - Attempt to initiate with non owner and check that it fails
 * - Initiate validator delisting
 * - Deliver the Warp message to the P-Chain (not implemented)
 * - Aggregate P-Chain signatures on the response Warp message
 * - Deliver the Warp message to the L1
 * - Verify that the validator is delisted from the validator manager contract
```
align comment with opening/closing `/*`